Makefile

```diff
@@ -155,10 +155,15 @@ integration-test: ## Run integration tests
 	@echo "==> Running integration tests..."
 	$(TEST_CMD) --tags="$(INTEGRATION_TAGS)" -count=1 ./internal...
 
+.PHONY: fuzz-normalizer-test
+fuzz-normalizer-test: ## Run fuzz test
+	@echo "==> Running fuzz test..."
+	$(TEST_CMD) -fuzz=Fuzz -fuzztime 50s --tags="$(UNIT_TAGS)" -race ./internal/kubernetes/operator/resources
```

Comment:
It's a part of unit tests. Go test -fuzz does not support multipackage testing. That's why I add a new command

Reply:
this is what evergreen runs
https://github.com/mongodb/mongodb-atlas-cli/blob/450429129d5d51837b7eb6f3558a185bfa8a4c0c/build/ci/evergreen.yml#L68-L75

which doesn't include `fuzz-normalizer-test` is this make target something you need to run on evergreen? I suppose you do

Reply:
I see. But this test is run in Github actions tests.

Reply:
up to you and the kube team, only evergreen tests are mandatory, we have GH actions mainly for easy in case of outside contributions given evergreen is restricted to only mongodb employees